tests/test_constructionset.py

```diff
@@ -0,0 +1,9 @@
+from app.models.energy.constructionset import WallSet, FloorSet, RoofCeilingSet, ApertureSet, DoorSet, ConstructionSet
```

Comment:
Let' just import what we are using in the module.

Also it looks like we are missing tests for each set of constructions.